plugins/inputs/ping/ping_windows_test.go

```diff
@@ -73,7 +73,8 @@ func TestPingGather(t *testing.T) {
 		pingHost: mockHostPinger,
 	}
 
-	acc.GatherError(p.Gather)
+	err := acc.GatherError(p.Gather)
+	require.NoError(t, err)
```

Comment:
A matter of taste but I prefer
```suggestion
	require.NoError(t, acc.GatherError(p.Gather))
```
Feel free to ignore. Same for other occurrences below.

Reply:
Changed